statistics/shortest_time_to_get_all_singles.rb

```diff
@@ -28,17 +32,11 @@ def transform(query_results)
       .map do |person_link, results|
         first_competition_date = results[0]["start_date"]
         first_successes = results
-          .select { |result| result["best"] > 0 && Events::OFFICIAL.include?(result["event_id"]) }
+          .select { |result| result["best"] > 0 }
           .group_by { |result| result["event_id"] }
           .map { |event_id, results| results.map { |result| result["start_date"] }.min }
-        days = if first_successes.length == Events::OFFICIAL.length
-                 (first_successes.max - first_competition_date).to_i
-               else
-                 nil
-               end
-        [person_link, days]
+        [(first_successes.max - first_competition_date).to_i, person_link]
```

Comment:
Given the SQL described before, we don't need this whole conditional statement.
Also changed the order of array elements to reflect the `@table_header` definition order.